{"uuid":"e947c9e2-5ef6-4350-9b5e-65d1f99f380f","historyId":"bbf591c19eb1cae67c3c05766dd0a9b1","fullName":"com.autotest.testcase.unittest.LoginApiTest.login","labels":[{"name":"package","value":"com.autotest.testcase.unittest.LoginApiTest"},{"name":"testClass","value":"com.autotest.testcase.unittest.LoginApiTest"},{"name":"testMethod","value":"login"},{"name":"parentSuite","value":"apiAutoTest"},{"name":"suite","value":"test"},{"name":"subSuite","value":"com.autotest.testcase.unittest.LoginApiTest"},{"name":"host","value":"xiaohe0128"},{"name":"thread","value":"27788@xiaohe0128.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"正反用例"},{"name":"epic","value":"接口单元测试"},{"name":"feature","value":"登录接口"}],"links":[],"name":"login","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.reflect.InvocationTargetException","trace":"java.lang.reflect.InvocationTargetException\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat com.autotest.testcase.unittest.LoginApiTest.reflect(LoginApiTest.java:29)\r\n\tat com.autotest.testcase.unittest.LoginApiTest.login(LoginApiTest.java:55)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:645)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:756)\r\n\tat org.testng.TestRunner.run(TestRunner.java:610)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1133)\r\n\tat org.testng.TestNG.run(TestNG.java:1104)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\nCaused by: java.lang.Exception: Failed : HTTP error code : 400\r\n\tat com.autotest.utils.RequestMethod.Post(RequestMethod.java:107)\r\n\t... 30 more\r\n"},"stage":"finished","description":"获取token令牌","steps":[],"attachments":[{"name":"请求地址：","source":"c228edb5-364b-436d-aa6c-5496081b8ac4-attachment.txt","type":"text/plain"},{"name":"请求方法","source":"72ba1dc4-e2ea-4619-83d6-93612647ba05-attachment.txt","type":"text/plain"},{"name":"请求参数","source":"528910ac-7494-41ad-8298-3f8d6d04b840-attachment.txt","type":"text/plain"},{"name":"请求头:","source":"68e726fa-6e5a-4abd-a8e8-07d0f022866f-attachment.txt","type":"text/plain"}],"parameters":[{"name":"arg2","value":"A8b52"},{"name":"arg1","value":"123"},{"name":"arg0","value":"admin"}],"start":1727181187669,"stop":1727181187685}